bom/application/pom.xml

```diff
@@ -138,7 +138,8 @@
         <brotli4j.version>1.16.0</brotli4j.version>
         <reactive-streams.version>1.0.4</reactive-streams.version>
         <jboss-logging.version>3.6.0.Final</jboss-logging.version>
-        <mutiny.version>2.6.0</mutiny.version>
+        <mutiny.version>2.6.1</mutiny.version>
+        <jctools-core.version>4.0.5</jctools-core.version>
```

Comment:
In the PR description you mention version 4.0.4. Is this on purpose or a typo?

Reply:
Good catch, that was a typo!